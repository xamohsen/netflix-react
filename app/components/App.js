import React, {useEffect} from "react";
import MainView from "./MainView";
import Navbar from "./Navbar";



const App = () => {

  return <div>
    <Navbar />
    <MainView testid={"home"} text={"Hello World"}/>
  </div>;
};

export default App;