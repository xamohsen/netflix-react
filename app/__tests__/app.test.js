import React from 'react'
import renderer from 'react-test-renderer';
import {
  render,
  waitForElement,
} from '@testing-library/react';
import App from "../components/App";

test('App component snapshot', () => {
  const component = renderer.create(<App/>);
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});

test('renders hello world text', async () => {
  const {getByText} = render(<App/>);
  await waitForElement(() => getByText(/Hello World/));
});