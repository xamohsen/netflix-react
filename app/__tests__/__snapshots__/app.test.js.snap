// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App component snapshot 1`] = `
<p>
  Hello World
</p>
`;
